kubernetes/resource_kubernetes_ingress_test.go

```diff
@@ -334,3 +356,70 @@ resource "kubernetes_ingress" "test" {
 	}
 }`, name)
 }
+
+func testAccKubernetesIngressConfig_waitForLoadBalancer(name string) string {
+	return fmt.Sprintf(`
+resource "kubernetes_service" "test" {
```

Comment:
For some reason this test wouldn't pass on gcloud if I didn't have a Service and Deployment behind the Ingress. I suspect because the load balancer needs an open port before it can become ready.